main.go

```diff
@@ -280,7 +280,7 @@ func writeKeyPair(name string, publicKey, privateKey [chacha20poly1305.KeySize]b
 	if err != nil {
 		return err
 	}
-	if format == "openssl" {
+	if format == "openssl" { //nolint
```

Comment:
I understand why we want to skip linting this one, later on we can refactor this function to simplify all these if-else statements

Reply:
yeah, i would like do this in a different PR, not to make this too big now.

Reply:
I re-arranged the code. hope that this is better now.